### PR TITLE
TFRS-2116 - Added filtering & sorting to the Compliance and Exclusion Reports page for columns -Original & Current Status, Last Updated On

### PR DESCRIPTION
**Pull request by vibhiquartech**

### CODE REVIEW

File: backend/api/viewsets/ComplianceReport.py [hunk 1]
```diff
@@ -241,7 +300,7 @@ def get_latest_supplemental_reports(self):
         latest_supplementals = ComplianceReport.objects.raw("""
             select distinct on (p.id) c.*
             from compliance_report p 
-              left join compliance_report c on p.id = c.supplements_id
+            left join compliance_report c on p.id = c.id
```
Comment (AlexZorkin) [inline backend/api/viewsets/ComplianceReport.py:303]: This seems odd. It's joining on itself and would only ever return one record. What was the intended purpose of this change?

Comment (AlexZorkin) [inline backend/api/viewsets/ComplianceReport.py:303]: This seems odd. It's joining on itself and would only ever return one record. What was the intended purpose of this change?

**Review by AlexZorkin — CHANGES_REQUESTED** [review]
just one question, otherwise looks good.

File: backend/api/viewsets/ComplianceReport.py (hunk 1 above)
Comment (vibhiquartech) [reply backend/api/viewsets/ComplianceReport.py]: Alex, I forgot to revert the changes while testing supplemental status column. I have reverted back the changes. Thanks for bringing that up

**Review by AlexZorkin — APPROVED** [review]

**Pull request merged**